modules/web/js/ballerina/ast/default-ballerina-ast-factory.js

```diff
@@ -254,14 +267,18 @@ DefaultBallerinaASTFactory.createAggregatedFunctionInvocationStatement = functio
 
         if (!_.isEmpty(args.functionDef.getReturnParams())) {
             // FIXME : Do a better solution to this by refactoring transform addChild and canDrop
-            const assignmentStmt = BallerinaASTFactory.createAssignmentStatement();
-            const leftOp = BallerinaASTFactory.createLeftOperandExpression(args);
-            const rightOp = BallerinaASTFactory.createRightOperandExpression(args);
-            rightOp.addChild(funcInvocationExpression);
-            assignmentStmt.addChild(leftOp);
-            assignmentStmt.addChild(rightOp);
-            assignmentStmt.accept(new EnableDefaultWSVisitor());
-            return assignmentStmt;
+            const variableDefinitionStatement = BallerinaASTFactory.createVariableDefinitionStatement();
+            let leftOperandExpression = '';
+            args.functionDef.getReturnParams().forEach((returnParam, index) => {
+                if (index === 0) {
+                    leftOperandExpression = returnParam.type + ' ' + (returnParam.identifier
+                            ? returnParam.identifier : returnParam.type.substr(0, 1));
+                }
+            });
+            const expression = leftOperandExpression + ' = ' + functionInvokeString;
```

Comment:
You can use string templating here if needed IMO. Lets try and use it in the future.